app/views/unattended/provisioning_templates/snippet/remote_execution_ssh_keys.erb

```diff
@@ -51,7 +51,7 @@ EOF
 
   chmod 0700 <%= ssh_path %>
   chmod 0600 <%= ssh_path %>/authorized_keys
-  chown -R <%= "#{ssh_user}:" %> <%= ssh_path %>
+  chown -R <%= "#{ssh_user}:" %> <%= "~#{ssh_user}" %>
```

Comment:
I get where you're going with this, but it would result in a slightly weird behavior if `ssh_path` was not inside that user's directory. It is a bit of an edge but but people but lots of odd things. How about we do both? Change the owner of the home directory and `ssh_path` as well just to be extra sure?

Reply:
Thanks @adamruzicka, for some reason I didn't receive an email notification when you commented. I've updated the pull request.